docs/src/pages/options/animations.md

```diff
@@ -41,7 +41,7 @@ If you are building a website, you can also skip configuring quasar.config.js an
 It should be noted that when you import Animate.css through the `<link>` tag, all animation CSS classes must be prefixed with `animate__`. This is a breaking change in the migration of Animate.css from v3 to v4. If you want to avoid using prefixes, you can import the [compat version](https://animate.style/#migration). However, if you're using the **Quasar CLI**, no additional changes are needed.
 :::
 
-::: info
+::: tip
```

Comment:
Not sure about the previous reasoning with the non-existent `info`, but I think `warning` suits better than `tip` here.
```suggestion
::: warning
```